### DISCUSSION TITLE
Multiple omp.cache files in Cygwin home directory when starting new shell (bash)

### DISCUSSION THREAD

**Discussion by wrecklass** [JanDeDobbeleer/oh-my-posh#5531] [Troubleshoot] [answered]
Title pretty much says it. I've noticed lately oh-my-posh keeps generating multiple omp.cache files in my user home  directory. I delete them, and  there are immediately several recreated immediately. In the past there might be one file generated every few days, but this is several in a few minutes. 

If they were either dotfiles or put in a dotfile where they weren't visible it would be great. Here is my latest directory after rebooting and starting a new bash shell.

```
>> ls -l
total 244
drwxrwx---+ 1 smartin smartin     0 May 18 10:21 Ark_ASA/
lrwxrwxrwx  1 smartin smartin    33 Jun 16 14:35 Desktop -> /c/Users/smartin/OneDrive/Desktop/
lrwxrwxrwx  1 smartin smartin    35 Jun 16 14:36 Documents -> /c/Users/smartin/OneDrive/Documents/
lrwxrwxrwx  1 smartin smartin    26 Jun 16 14:36 Downloads -> /c/Users/smartin/Downloads/
lrwxrwxrwx  1 smartin smartin    26 Jun 16 14:36 Nextcloud -> /c/Users/smartin/Nextcloud/
drwxrwx---+ 1 smartin smartin     0 Aug 27 02:47 bin/
-rwxrwx---+ 1 smartin smartin   196 May 25 01:42 gzw_ip.sh*
drwxrwx---+ 1 smartin smartin     0 May 26 02:42 help/
-rwxrwx---+ 1 smartin smartin   302 Aug 27 00:23 omp.cache*
-rwxrwx---+ 1 smartin smartin     2 Aug 27 00:19 omp.cache.10140*
-rwxrwx---+ 1 smartin smartin     2 Aug 27 00:16 omp.cache.11836*
-rwxrwx---+ 1 smartin smartin 13569 Aug 27 00:09 omp.cache.2337*
-rwxrwx---+ 1 smartin smartin 11300 Aug 27 00:16 omp.cache.2477*
-rwxrwx---+ 1 smartin smartin 13574 Aug 27 00:18 omp.cache.2534*
-rwxrwx---+ 1 smartin smartin 13577 Aug 27 00:20 omp.cache.2702*
-rwxrwx---+ 1 smartin smartin 13581 Aug 27 00:23 omp.cache.2785*
-rwxrwx---+ 1 smartin smartin 13208 Aug 27 00:50 omp.cache.2839*
-rwxrwx---+ 1 smartin smartin     2 Aug 27 00:20 omp.cache.28504*
-rwxrwx---+ 1 smartin smartin     2 Aug 27 00:47 omp.cache.30564*
-rwxrwx---+ 1 smartin smartin     2 Aug 27 00:17 omp.cache.8868*
-rwxr--r--+ 1 smartin smartin    54 Aug 26 23:57 rasp_rack.txt*
```
Several are just a pair of empty braces: '{}'

Any way to redirect these? An environment variable would be great. Unfortunately the XDG_CACHE_HOME variable makes no difference, I suspect because OMP knows it's not on Linux.

**Comment by JanDeDobbeleer** [comment]
@wrecklass on Windows it expects `LOCALAPPDATA/oh-my-posh`, I'll see if I can also add `OMP_CACHE_DIR` to the equation.

**Reply by JanDeDobbeleer** [reply, marked as answer]
solved with #5532

**Reply by wrecklass** [reply]
I'm not changing things after the new update to see how that works out. LOCALLAPPDATA is definitely set, and I see the the LOCALLAPPDATA/oh-my-posh is there and has about 50 omp.cache.xxxxx files dating back to July. Once cleared most of them out and will monitor for a few days before setting a new variable. 

BTW I noticed that under the above folder is another oh-my-posh dir. So: LOCALLAPPDATA/oh-my-posh/oh-my-posh/! Which also has a couple of very recent omp.cache files as well. 😀
